src/pages/sql/sql-interview-questions/index.md

```diff
@@ -51,8 +51,62 @@ NOT NULL is the only constraint that works at the column level.
 A pseudocolumn is a function which returns a system generated value. The reason it is known as so because a pseudocolumn is an Oracle assigned value used in the same context as an Oracle database column but not stored on disk.
 ``` Some examples of it are:
     ROWNUM, ROWID, USER, CURRVAL, NEXTVAL etc.
-```    
+```   
 
+# Changes2
```

Comment:
Hi @anwesha999 

Can you please change this heading to something more appropriate than `Changes2`? You can make the heading something that relates to the question that is being answered.